frontend/src/App.css

```
@@ -2829,917 +2746,128 @@ code {
-.ops-card-title:hover {
-  color: var(--color-primary);
-}
+/* ============================================================
+   NTD Logo — 金色流光效果
+   ============================================================ */
 
-.ops-status-icon {
-  font-size: 16px;
+.ntd-logo {
+  position: relative;
+  font-family: 'Georgia', 'Times New Roman', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Georgia" (font-family-name-quotes)

(font-family-name-quotes)